.github/workflows/fluent-bit-coralogix-chart.yml

```diff
@@ -5,10 +5,11 @@ on:
   push:
     branches: master
     paths:
-    - 'fluent-bit/coralogix/**'
+    - 'logs/fluent-bit/http/Chart.yaml'
+    - 'logs/fluent-bit/http/values.yaml'
```

Comment:
this workflow is specific for fluentbit-coralogix and not http, so the paths need to be '/logs/fluent-bit/coralogix' , also I would keep it like the origin with **

Reply:
we can add:
```
    paths-ignore: 
          - '**.md'
```
or something like this:
```
    paths:
      - 'logs/fluent-bit/http/**'
      - '!logs/fluent-bit/http/README.md**'
```
so it will ignore the readme, but everything else in the chart should trigger it, I dont see a reason why not : )

Reply:
took your advice, used paths-ignore